scene/3d/physics_body.cpp

```diff
@@ -92,17 +136,42 @@ void PhysicsBody::remove_collision_exception_with(Node* p_node) {
 	PhysicsServer::get_singleton()->body_remove_collision_exception(get_rid(),physics_body->get_rid());
 }
 
+void PhysicsBody::_set_layers(uint32_t p_mask) {
```

Comment:
No, since it is `private`, and as such can't be used from scripts (AFAIK).

Also, we don't want a print every time an old scene is opened by the new version, do we?